src/cloudkit.cs

```diff
@@ -350,6 +555,52 @@ interface CKFetchRecordChangesOperation {
 			set; 
 		}
 	}
+	
+	[iOS (10,0), Watch (3,0), TV (10,0), Mac (12,10, onlyOn64 : true)]
+	[BaseType (typeof(CKDatabaseOperation))]
+	interface CKFetchRecordZoneChangesOperation
+	{
+		[Export ("initWithRecordZoneIDs:optionsByRecordZoneID:")]
+		IntPtr Constructor (CKRecordZoneID[] recordZoneIDs, [NullAllowed] NSDictionary<CKRecordZoneID, CKFetchRecordZoneChangesOptions> optionsByRecordZoneID);
+
+		[Export ("recordZoneIDs", ArgumentSemantic.Copy)]
+		CKRecordZoneID[] RecordZoneIds { get; set; }
+
+		[NullAllowed, Export ("optionsByRecordZoneID", ArgumentSemantic.Copy)]
+		NSDictionary<CKRecordZoneID, CKFetchRecordZoneChangesOptions> OptionsByRecordZoneId { get; set; }
```

Comment:
Got to say, I did wrongly use ID in some of them. Let me know and I'll change the code to the desired style.